src/transform.ts

```diff
@@ -3,6 +3,13 @@ import swc from "../lib/wasm.js";
 
 const DEFAULT_OPTIONS = {
 	mode: "strip-only",
+	// default transform will only work when mode is "transform"
+	transform: {
+		verbatimModuleSyntax: true,
+		nativeClassProperties: true,
+		noEmptyExport: true,
+		importNotUsedAsValues: "preserve",
+	},
```

Comment:
LGTM.

I think the only behavioral change this PR will achieve on Node is to fix https://github.com/nodejs/node/issues/54514 via the addition of `noEmptyExport: true`

The rest is just codifying existing usage.  Similarly you could add:

```suggestion
                useDefineForClassFields: true,
	},
```

In TypeScript, VMS replaces that need to set INUAV.  But I don't know if that's the case in SWC so keeping it fully explicit (as you have now) until we hear otherwise from SWC folk seems good.

Reply:
apparently `useDefineForClassFields` is not in the allowed configuration